src/providers/ldap/sdap_async.c

```diff
@@ -2416,7 +2417,8 @@ static void sdap_sd_search_done(struct tevent_req *subreq)
     talloc_zfree(subreq);
     if (ret != EOK) {
         DEBUG(SSSDBG_OP_FAILURE,
-              "sdap_get_generic_ext_recv failed [%d]: %s\n",
+              "sdap_get_generic_ext_recv failed [%d]: %s "
```

Comment:
`sdap_get_generic_ext_recv()` doesn't always fail due to a timeout, so I think that the debug message should differ depending on the error.